doc/en/components/grids/combo/single-selection.md

```diff
@@ -0,0 +1,83 @@
+---
+title: {Platform} ComboBox Component – {ProductName}
+_description: {Platform} Combo component provides a powerful input, combining features of the basic HTML input, select, filtering and custom drop-down lists. Try it for FREE
+_keywords: {ProductName}, UI controls, {Platform} widgets, web widgets, UI widgets, {Platform}, Native {Platform} Components Suite, Native {Platform} Controls, Native {Platform} Components Library, {Platform} ComboBox component
+mentionedTypes: ['Combo', 'Single Selection Combo', 'ComboItem', 'ComboHeader', 'ComboList']
+---
+
+# {Platform} Single Selection ComboBox
+
+The {Platform} `ComboBox` supports single-selection mode and quick filtering of the list of items via the main input prompt. Users can quickly type in the item they are looking for and be presented with a list of options. Upon pressing the enter key, the first highlighted match will be selected.
+
+## {Platform} Single Selectoin Example
+
+To enable single-selection and quick filtering, set the `SingleSelect` property on the `ComboBox` component. The user experience and keyboard navigation will mostly stay the same, but instead of having to type in your search query into a special filtering box above the list of options, the main input box will be used.
+
+```html
+<igc-combo single-select></igc-combo>
+```
+
+<code-view style="height:320px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/grids/combo-simplified"
+           alt="{Platform} Single Selection Combo Example"
+           github-src="grids/combo/simplified">
```

Comment:
please create this sample in WC repo
https://github.com/IgniteUI/igniteui-wc-examples/tree/vnext/samples/grids/combo

Reply:
It already has been created.
https://github.com/IgniteUI/igniteui-wc-examples/pull/366